examples/analyze-bundles/package.json

```diff
@@ -5,7 +5,8 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "analyze": "cross-env ANALYZE=true yarn build"
+    "analyze": "cross-env ANALYZE=true yarn build",
+    "serve": "serve .next/analyze"
```

Comment:
I have added a `serve` script so that you can easily serve the output of the analyzer.